requirements/travis.txt

```diff
@@ -8,7 +8,7 @@ appdirs==1.4.4            # via virtualenv
 certifi==2020.6.20        # via requests
 chardet==3.0.4            # via requests
 codecov==2.1.7            # via -r requirements/travis.in
-coverage==5.2             # via codecov
+coverage==5.1             # via codecov
```

Comment:
A version downgrade?

Reply:
This is pretty weird. This is either pip-compile tricks or something happened while resolving conflicts that were in dependencies. Anyway, there is now `coverage == 5.2`